tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -164,9 +164,9 @@ source "${VENV_DIR}/bin/activate" || \
 
 
 # Install the pip file in virtual env
-pip install -v --force-reinstall ${WHL_PATH} \
-&& echo "Successfully installed pip package ${WHL_PATH}" \
-|| die "pip install (without --upgrade) FAILED"
+pip install -v --upgrade --no-deps --force-reinstall ${WHL_PATH} || \
+    die "pip install (without --upgrade) FAILED"
```

Comment:
right, the comment should change :)

I'm afraid the upgrade and no-deps has to stay both otherwise we would upgrade all the dependencies. And we do not want that.

I'm going to update this PR a bit...